docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1274 @@
+    "        ax.axis(\"off\")\n",
+    "\n",
+    "    plt.tight_layout(h_pad=2.0)\n",
+    "    plt.show()\n",
+    "```\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "def plot_image_issue_examples(issues_df, num_examples=15):\n",
+    "    ncols = 5\n",
+    "    nrows = int(math.ceil(num_examples / ncols))\n",
+    "\n",
+    "    _, axes = plt.subplots(nrows=nrows, ncols=ncols)\n",
+    "    axes_list = axes.flatten()\n",
+    "    issue_indices = issues_df.index.values\n",
+    "\n",
+    "    for i, ax in enumerate(axes_list):\n",
+    "        if i >= num_examples:\n",
+    "            ax.axis(\"off\")\n",
+    "            continue\n",
+    "        idx = int(issue_indices[i])\n",
+    "        label = label_issues.loc[idx][\"given_label\"]\n",
+    "        predicted_label = label_issues.loc[idx][\"predicted_label\"]\n",
+    "        ax.set_title(\n",
+    "            f\"id: {idx}\\n GL: {label}\\n SL: {predicted_label}\",\n",
+    "            fontdict={\"fontsize\": 8},\n",
+    "        )\n",
+    "        ax.imshow(dataset[idx][\"image\"], cmap=\"gray\")\n",
+    "        ax.axis(\"off\")\n",
+    "\n",
+    "    plt.tight_layout(h_pad=2.0)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_image_issue_examples(dark_issues_df, num_examples=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see from above examples that too dark images can also lead to label errors as it is difficult to see the contents of the image clearly."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View low information images\n",
+    "\n",
+    "In this section, we display examples from the dataset that are low information. The `lowinfo_issues` DataFrame tells use which examples are considered to be low information, and we can sort them via the `low_information_score` which quantifies how severe this issue is for each image (lower values indicate more severe intances of a type of issue).\n",
+    "\n",
+    "This allows us to visualize examples in the dataset which are most severe examples of low information issue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lowinfo_issues = lab.get_issues(\"low_information\")\n",
+    "lowinfo_issues_df = lowinfo_issues.query(\"is_low_information_issue\").sort_values(\n",
+    "    \"low_information_score\"\n",
+    ")\n",
+    "lowinfo_issues_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_image_issue_examples(lowinfo_issues_df, num_examples=5)"
+   ]
+  }
```

Comment:
Notebook needs to have a hidden cell at the end that serves as a unit test that this whole notebook was outputting good results. Look at the other notebooks.

Remember this notebook is constantly being re-run in CI, and we need to be sure it's always building with good results. Only way to do that is with a final cell that runs some sort of asserts

Reply:
Added a test cell